pkg/detector/detector.go

```diff
@@ -1141,32 +1091,27 @@ func (d *ResourceDetector) HandleClusterPropagationPolicyDeletion(policyID strin
 		policyv1alpha1.ClusterPropagationPolicyPermanentIDLabel: policyID,
 	}
 
-	cleanupMarksFun := func(obj metav1.Object) {
-		util.RemoveLabels(obj, clusterPropagationPolicyMarkedLabels...)
-		util.RemoveAnnotations(obj, clusterPropagationPolicyMarkedAnnotations...)
-	}
-
 	// load the ClusterResourceBindings which labeled with current policy
 	crbs, err := helper.GetClusterResourceBindings(d.Client, labelSet)
 	if err != nil {
 		klog.Errorf("Failed to list clusterResourceBindings with clusterPropagationPolicy permanentID(%s), error: %v", policyID, err)
 		errs = append(errs, err)
 	} else if len(crbs.Items) > 0 {
 		for index, binding := range crbs.Items {
-			// Must remove the marks, such as labels and annotations, from the resource template ahead of
+			// Must remove the claim metadata, such as labels and annotations, from the resource template ahead of
```

Comment:
claimed?

Reply:
'Claimed metadata' refers to metadata that has already been marked or declared, 'claim metadata' refers to metadata that is intended for marking or declaring purposes. When describing metadata that is used for marking or declaring resources, 'claim metadata' might be more appropriate.